src/errors.js

```diff
@@ -7,3 +7,36 @@ import { createCustomErrorClass } from "@ledgerhq/errors";
 export const TransactionRefusedOnDevice = createCustomErrorClass(
   "TransactionRefusedOnDevice"
 );
+
+export const ModeNotSupported = createCustomErrorClass("ModeNotSupported");
```

Comment:
this can be an invariant because it is a programming error, will never be a user error.

Reply:
It's already an invariant, I just forgot to remove this unused error :D